### DISCUSSION TITLE
Streaming chart currently not working?

### DISCUSSION THREAD

**Discussion by herzplatinedurchgebrannt** [Megabit/Blazorise#4323] [Q&A]
Hey guys,

I am working on an blazor server application with .net6, which is planned to run on a raspberry pi4.

I have a problem with the streaming chart: 
The chart itself is displayed, but there is no graph. The y-axis values change and this axis constantly rescales.

https://blazorise.com/docs/extensions/chart-live

I was confused by that, and then I have seen, on your homepage I have the same issue.
My dev machine is an Win10 notebook, the browser is Edge.

The standard charts (without streaming) work without issues.

Is this a bug?

Thank you in advance!
Cheers,
Alex

**Comment by stsrki** [comment]
That's weird. Something is definitely broken here. I will look into it.

**Reply by stsrki** [reply]
I don't know why it suddenly doesn't work with the default .NET settings. But adding `MaximumReceiveMessageSize` into Startup.cs seems to be working.

```diff
services.AddRazorPages();
services.AddServerSideBlazor();
services.AddHttpContextAccessor();

+services.AddServerSideBlazor().AddHubOptions( ( o ) =>
+{
+    o.MaximumReceiveMessageSize = 1024 * 1024 * 100;
+} );
```

Please try and see if it works for you. I will also add a note to the charts page saying the same.

**Reply by herzplatinedurchgebrannt** [reply]
I implemented your code into my Program.cs (in current blazor template there is no Startup.cs anymore), but still the same issue.

On my dev machine (ASPNETCORE_ENVIRONMENT: Production) it works, but still no success with the raspberry pi

**Reply by stsrki** [reply]
That would be hard to reproduce on my side 😅
